tests/test_result_frame_aggregator.py

```diff
@@ -345,10 +345,10 @@ def test_max_aggregation(self, res1d_river_network, column_mode):
         df_agg = df_agg.groupby("name").agg("max")  # test for entire reach (not segments)
 
         assert list(df_agg.columns.values) == [
+            "max_WaterLevel",
```

Comment:
What is the reason to shuffle this around?

Reply:
The new stack algorithm used in future pandas has a different order than before unfortunately